mlflow/pyspark/ml/__init__.py

```diff
@@ -111,27 +132,42 @@ def _get_instance_param_map(instance):
         if instance.isDefined(param)
     }
     expanded_param_map = {}
+
+    is_pipeline = isinstance(instance, Pipeline)
+    is_parameter_search_estimator = isinstance(instance, (CrossValidator, TrainValidationSplit))
+
     for k, v in param_map.items():
```

Comment:
```suggestion
    for param_name, param_value in param_map.items():
```

Can we rename these variables for better readability?